packages/synapse-react-client/src/components/StandaloneQueryWrapper/StandaloneQueryWrapper.tsx

```diff
@@ -100,6 +106,9 @@ function StandaloneQueryWrapper(props: StandaloneQueryWrapperProps) {
     noContentPlaceholderType = showTopLevelControls
       ? NoContentPlaceholderType.INTERACTIVE
       : NoContentPlaceholderType.STATIC,
+    cardConfiguration,
```

Comment:
Previously you could pass both a cardConfiguration and tableConfiguration and this would happily render both.  Now it does not render the table if the card configuration is available.